core/src/test/java/org/apache/accumulo/core/file/blockfile/cache/TestCachedBlockQueue.java

```diff
@@ -58,34 +106,38 @@ public void testQueue() {
     long expectedSize = cb1.heapSize() + cb2.heapSize() + cb3.heapSize() + cb4.heapSize()
         + cb5.heapSize() + cb6.heapSize() + cb7.heapSize() + cb8.heapSize();
 
-    assertEquals(queue.heapSize(), expectedSize);
+    assertEquals(expectedSize, queue.heapSize());
```

Comment:
Would be nice to do some checks on `sum`.  Could add the following check, I think its correct but not 100% sure.
```suggestion
    assertEquals(expectedSize, queue.heapSize());
     assertEquals(expectedSize,sum.get());
```

Reply:
This instance is testing eviction, so the sum is the total blocks added, not the size contained in the cache.  Will add 
```
assertEquals(expectedSize, sum.get() - cb9.heapSize() - cb10.heapSize());
```
which removes the size of the blocks that are not expected to be included.

Reply:
Addressed in d0b5f0d3cb